templates/common/base.html

```diff
@@ -80,7 +80,8 @@ <h1 class="hnname"><a href="/">DataTau</a></h1>
               <a href="https://thegurus.tech">Made with ❤️ by The Gurus</a> |
               <a href="mailto:datatau@thegurus.tech">Support</a> |
               <a href="mailto:datatau@thegurus.tech">Contact</a> |
-              <a href="{% url 'latest_feed' %}">RSS</a>
+              <a href="{% url 'latest_feed' %}">RSS</a> |
+              <a rel="noopener noreferrer" target="__blank" href="https://twitter.com/datatau_net">Twitter</a>
```

Comment:
I see that the value of target is `__blank` (two underscores). Is that intentional? AFAIK, it's supposed to be `_blank` (one underscore).